src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -81,13 +85,23 @@
         return ResponseEntity.ok().body(networkModificationService.getModificationGroups());
     }
 
-    @PutMapping(value = "/networks/{networkUuid}/lines/{lineId}/status", consumes = MediaType.TEXT_PLAIN_VALUE)
+    @PutMapping(value = "/networks/{networkUuid}/group/{groupUuid}/lines/{lineId}/status", consumes = MediaType.TEXT_PLAIN_VALUE)
```

Comment:
API looks strange, groups is not a sub resource of network. 
Not easy to design but I think we should choose if the primary (I means most important) is network or modification group and then specify the the second infos with query param.
Like /networks/{networkUuid}/lines/{lineId}/status&group={groupUuid}.
Maybe in some use case the group is not necessary as we might (not for gridsuite) be able to modify the network without recording the modification.

Reply:
Done